content/using-sagemaker-managed-spot-training/notebooks/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Running Notebooks"
+chapter: false
+weight: 20
+---
+
+The GitHub Repository cloned to the Amazon SageMaker Notebook Instance contains sample notebooks that demonstrate leveraging Managed Spot Training with various frameworks. As new sample notebooks are developed, they will be added to the GitHub Repository located here: [EC2 Spot Labs](https://github.com/awslabs/ec2-spot-labs). 
+
+The general steps for executing notebooks are as follows:
```

Comment:
is there justification for these instructions seeing as you're providing the link to the notebook in the next step?

Reply:
I included it in case someone was more familiar with Jupyter Notebooks and wanted a TLDR overview of the workshop. Not sure if there's a better way to handle that, but for some folks they may want to just get in and run Notebooks.